.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     project: './tsconfig.json',
     ecmaVersion: 2020,
     sourceType: 'module',
+    tsconfigRootDir: __dirname,
```

Comment:
i only opened web folder in my vs code so i think .eslintrc.js treated it as external directory

Reply:
> Hi @MuzzammilRafiq, a small question, how did you experience this error? Because I've never encountered that. Did you open the project using VSCODE from the root? Or any other way?

I experience this error when not using VS Code. I had it fixed in my branch that fixes Eslint React Hooks rules settings but is far from be merged.